release-notes/release-notes-template.md

```diff
@@ -0,0 +1,55 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud < release-version >.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Feature 1
+
+< Explain feature 1 >
+
+### Feature 2
+
+< Explain feature 2 >
+
+
+< Add feature sections as necessary >
+
+
+### Dashboard improvements
+
+< List enhancements to the dashboard here. You can screenshots to illustrate something new or enhanced. >
+
+
+### Other
+
+< In addition to the below placeholder points, list minor features that are part of the release but were not elaborated as separate sections >
+
+* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
+* The Android WebView has been updated to < insert-version-along-with-link >.
+
+
+### Removed functionality
+
+< List features that were part of Anbox Cloud so far or that were on the deprecation path for a while and have been removed for the current release >
+
+
+### Deprecations
```

Comment:
Same as above: should this be a subsection of `New features & improvements` or should it be its own section?

Reply:
See above answer

Reply:
```suggestion
## Deprecations
```